### PR TITLE
Link Record Unit Tests

### PR DESCRIPTION
**Pull request by Blackbaud-CaseyKour**

### CODE REVIEW

File: src/app/public/src/modules/link-records/link-records.component.spec.ts [hunk 1]
```diff
@@ -3,158 +3,314 @@ import {
...
-        imports: [
-          SkyContribLinkRecordsModule
-        ]
+    component.items = Observable.of([item]);
+    component.matches = Observable.of([linkRecordMatch]);
+
+    fixture.detectChanges();
+
+    component.records.take(1)
+      .subscribe(r => {
```
Comment (Blackbaud-CaseyKour) [inline src/app/public/src/modules/link-records/link-records.component.spec.ts]: I get a linting error on this particular line here: "Parameter r has an implicit any type". Does this need to be resolved?

Comment (Blackbaud-LuisBello) [reply src/app/public/src/modules/link-records/link-records.component.spec.ts]: yes, all lint errors need to be resolved, there should be 0

**Comment by blackbaud-joshgerdes** [comment]
also please adjust fixture names to not use camel case.

**Comment by Blackbaud-CaseyKour** [comment]
Need to cover all branch cases

**Pull request closed without merging**